cmd/cluster/kubevirt/create.go

```diff
@@ -0,0 +1,88 @@
+package kubevirt
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/spf13/cobra"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "kubevirt",
+		Short:        "Creates basic functional HostedCluster resources for KubeVirt platform",
+		SilenceUsage: true,
+	}
+
+	opts.KubevirtPlatform = core.KubevirtPlatformCreateOptions{
+		Memory:             "4Gi",
+		Cores:              2,
+		ContainerDiskImage: "",
+	}
+
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.Memory, "memory", opts.KubevirtPlatform.Memory, "The amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
+	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.Cores, "cores", opts.KubevirtPlatform.Cores, "The number of cores inside the vmi, Must be a value greater or equal 1")
```

Comment:
>"The number of cores inside the vmi, Must be a value greater or equal 1

Should we validate this below and fail otherwise? the way it's now zero would pass through right?

Reply:
Added validation in `applyPlatformSpecificsValues` function